LIPs/LIP-36.md

```diff
@@ -87,22 +92,25 @@ Add the following steps to the algorithm for `bondingManager.updateTranscoderWit
     - Set `earningsPool.cumulativeFeeFactor = prevEarningsPool.cumulativeFeeFactor + prevEarningsPool.cumulativeRewardFactor * (delegatorsFees / X)`
 - If `prevEarningsPool.cumulativeFeeFactor > 0`:
     - Set `earningsPool.cumulativeFeeFactor += prevEarningsPool.cumulativeRewardFactor * (delegatorFees / X)` 
+- If the orchestrator's `lastFeeRound` is smaller than the round the current fees are being added for set `lastFeeRound` to that round. 
```

Comment:
Let's add `lastFeeRound` to the data types section since this is an additional storage variable included in the LIP.

Reply:
fixed in 4823679